src/main/kotlin/domain/MineGenerator.kt

```diff
@@ -0,0 +1,29 @@
+package domain
+
+import kotlin.random.Random
+
+object MineGenerator {
+    private val lists: MutableList<Location> = mutableListOf()
+    fun create(height: Int, width: Int, count: Int): List<Location> {
+        for (i in 1..count) {
+            lists.add(getNewLocation(height, width))
+        }
+        return lists
+    }
```

Comment:
MineGenerator 를 재활용하기 위해선 지역변수로 선언하는것이 어떨까요 ? 

```suggestion
    fun create(height: Int, width: Int, count: Int): List<Location> {
        val lists: MutableList<Location> = mutableListOf()
        for (i in 1..count) {
            lists.add(getNewLocation(height, width))
        }
        return lists
    }
```